pkgs/applications/science/logic/coq/default.nix

```diff
@@ -48,27 +48,28 @@ let
   releaseRev = v: "V${v}";
   fetched = import ../../../../build-support/coq/meta-fetch/default.nix
     { inherit lib stdenv fetchzip; }
-    { inherit release releaseRev; location = { owner = "coq"; repo = "coq";}; }
-    args.version;
-  version = fetched.version;
-  coq-version = args.coq-version or (if version != "dev" then versions.majorMinor version else "dev");
-  versionAtLeast = v: (coq-version == "dev") || (lib.versionAtLeast coq-version v);
-  ideFlags = optionalString (buildIde && !versionAtLeast "8.10")
+    { inherit release releaseRev; combined = true;
+      location = { owner = "coq"; repo = "coq";}; }
+    { inherit origin version; pname = "coq"; };
+  coq-version = args.coq-version or (versions.majorMinor fetched.version);
+  coqAtLeast = lib.versionAtLeast coq-version;
```

Comment:
`coqAtLeast` should probably get documented as well.

Reply:
actually it is local to coq/default.nix so I'm not sure it should...